### PR TITLE
Update framework + use EquilateralTriangle in Triangles.

### PR DESCRIPTION
**Pull request by smoogipoo**

### CODE REVIEW

File: osu.Game/Graphics/Backgrounds/Triangles.cs [hunk 1]
```diff
@@ -99,16 +99,18 @@ protected virtual Triangle CreateTriangle()
 
             const float size = 100;
 
-            return new Triangle
+            return new EquilateralTriangle
             {
                 Origin = Anchor.TopCentre,
+
```
Comment (peppy) [inline osu.Game/Graphics/Backgrounds/Triangles.cs]: are these newlines on purpose? o_O

Comment (smoogipooo) [inline osu.Game/Graphics/Backgrounds/Triangles.cs]: Yup, I like to split these declarations into the following:
1) Origins/Anchors
2) Sizing/positioning
3) Other single-line properties (colour, border, depth, etc).
4) Multi-line properties (edgeeffect)
5) Children =...

Just helps with readability imo, so I don't personally enforce it.

Comment (peppy) [inline osu.Game/Graphics/Backgrounds/Triangles.cs]: that's fine and all, but we're also going for a consistent coding style (and this is inconsistent at the moment). maybe put this up for discussion and we can do a pass on the whole codebase, but don't use this for now?

Comment (smoogipooo) [inline osu.Game/Graphics/Backgrounds/Triangles.cs]: Wouldn't say this is inconsistent at all in that case. Just that there's no guideline or style for this.

Can put a discussion up, sure, but I'm not changing this right now unless it's enforced or there's already currently a guideline - I see it as a waste of time. Especially considering taiko is coded strictly this way.

**Pull request merged**